dist/images/ovnkube.sh

```diff
@@ -498,16 +498,16 @@ cleanup-ovs-server () {
   /usr/share/openvswitch/scripts/ovs-ctl stop
 }
 
-# create the ovnkube_db endpoint for other pods to query the OVN DB IP
-create_ovnkube_db_ep () {
+# set the ovnkube_db endpoint for other pods to query the OVN DB IP
+set_ovnkube_db_ep () {
   # delete any endpoint by name ovnkube-db
   kubectl --server=${K8S_APISERVER} --token=${k8s_token} --certificate-authority=${K8S_CACERT} \
     delete ep -n ${ovn_kubernetes_namespace} ovnkube-db 2>/dev/null
 
```

Comment:
@squeed @dcbw but we delete the endpoint before we create, so how does it fail with `already exists` error?

Reply:
Because I was hacking on an older version of the script and didn't notice this. Good point.

We should probably just remove the `delete`.

Reply:
@dcbw @squeed  ok.